imports/ui/pages/professor/run_session.jsx

```diff
@@ -373,15 +381,11 @@ class _RunSession extends Component {
 
           <Tooltip
             position='bottom'
-            interactive
+            interactive='true'
+            theme='light'
+            hideOnClick='true'
             html={
-              <div>
-                {students.map(student =>
-                  (<div key={student._id}>
-                    {student.profile.lastname + ', ' + student.profile.firstname}
-                  </div>)
-                )}
-              </div>
+              <StudentListTooltip students={students} toggleProfileViewModal={this.toggleProfileViewModal} />
```

Comment:
I wonder if it would be better to have students (line 366) passed in as a prop to the _RunSession component, since we should avoid db queries in the React components. As is, I would think that the list of students is not reactive. That is, if prof starts the session, then student joins and prof does not refresh screen, does the list of students update?

Reply:
The list of students does update, but I agree that we should avoid db queries in react components